app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    # user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
I would try to avoid leaving commented out code in when pushing to the master

Reply:
Good spot! I'm temporarily leaving these in as they will be used for logging in (when I get to it).